src/Tizen.NUI/src/public/BaseComponents/Style/Selector.cs

```diff
@@ -33,11 +33,18 @@ public class Selector<T>
     {
         private readonly bool cloneable = typeof(ICloneable).IsAssignableFrom(typeof(T));
 
+        private IList<StateValuePair<T>> stateValueList = new List<StateValuePair<T>>();
+
         /// <summary>
         /// The list for adding state-value pair.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public IList<StateValuePair<T>> StateValueList { get; set; } = new List<StateValuePair<T>>();
```

Comment:
Oh, `public` is temporarily added to fix DA build error. (https://github.com/Samsung/TizenFX/pull/2142)

You don't need to remove setter of `StateValueList`.

Reply:
Thank you for your comment!
I have restored the code.